CONTRIBUTING.md

```diff
@@ -27,6 +27,7 @@ The localization process is defined below:
 - Japanese: [@kaakaa](https://github.com/kaakaa)
 - Polish: [@burasuk](https://github.com/burasuk)
 - Spanish: [@ernestoittig](https://github.com/ernestoittig)
+- Chinese: [@sharuru](https://github.com/sharuru)
```

Comment:
There are two kinds of Chinese (`zh-CN` and `zh-TW`) in Mattermost, so I think that it has better to clarify which Chinese have been translated. According to [Mattermost docs](https://docs.mattermost.com/developer/localization.html#translation-maintenance), `zh-CN` is called `Simplified Chinese`, so what do you think about rewriting this and README to `Simplified Chinese`?

Reply:
Oh, it's my mistake.
`Simplified Chinese` would be better.